include/logging/log_core.h

```diff
@@ -895,6 +896,31 @@ do {									       \
 
 #include <syscalls/log_core.h>
 
+/** @brief Function pointer table for log output formats, indexed by enum log_format_flags.
+ *
+ */
+extern void (*log_format_table[])(const struct log_output*, struct log_msg2*, uint32_t);
+
+/** @brief Set logging format.
+ *
+ * @param backend_name Backend Name Example: log_backend_uart.
```

Comment:
Name of the backend. Example: "log_backend_uart"

(The period helps avoid potential ambiguity.)

Reply:
Modified the doxygen. Thanks!!